pmd-cataloger/build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 plugins {
   java
   application
+  jacoco
```

Comment:
Let's set this to a specific version.

Reply:
@jbartolotta-sfdc , done. I've set it to 0.8.7.